File: agent/testAgent/evals/fixtures/f3-green/src/calc.ts
// calc.ts —— 数值工具

export function sum(nums: number[]): number {
  let total = 0;
  for (const n of nums) {
    total += n;
  }
  return total;
}

export function avg(nums: number[]): number {
  return sum(nums) / nums.length;
}

File: testAgent/evals/fixtures/f2-maven-failing/src/main/java/com/eval/Calc.java
package com.eval;

// Calc.java —— 数值工具
public class Calc {
    public static int sum(int[] nums) {
        int total = 0;
        for (int i = 0; i < nums.length - 1; i++) {
            total += nums[i];
        }
        return total;
    }
}
